packages/libraries/linkable-specs/src/link-import.ts

```
@@ -0,0 +1,40 @@
+    public as: string | null,
+  ) {}
+
+  public toString(): string {
+    return this.as ? `{ name: "${this.name}", as: "${this.as}" }` : `"${this.name}"`;
+  }
+
+  static fromTypedefs(node: ConstValueNode): FederatedLinkImport[] {
+    if (node.kind == Kind.LIST) {
+      const imports = node.values.map(v => {
+        if (v.kind === Kind.STRING) {
+          return new this(v.value, null);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)